### PR TITLE
ffplay: add more examples

### PR DESCRIPTION
**Pull request by Managor**
Add nodisp option and an example of piping into ffplay

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/ffplay.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 
 `ffplay {{path/to/file}}`
 
+- Play audio from a media file without a GUI:
+
+`ffplay -nodisp {{path/to/file}}`
+
+- Play media passed by ffmpeg through standard stream:
```
Comment (kbdharun) [inline pages/common/ffplay.md:14]:
```suggestion
- Play media passed by `ffmpeg` through standard stream:
```

Comment (Managor) [reply pages/common/ffplay.md:14]: That doesn't convey well enough that it supports protocols in my opinion. Like rtsp://

**Review by kbdharun — APPROVED** [review]

File: pages/common/ffplay.md [hunk 2]
```diff
@@ -7,6 +7,14 @@
 
 `ffplay {{path/to/file}}`
 
+- Play audio from a media file without a GUI:
+
+`ffplay -nodisp {{path/to/file}}`
+
+- Play media passed by `ffmpeg` through standard stream:
+
+`ffmpeg -i {{path/to/file}} -c {{copy}} -f {{mediaformat}} - | ffplay -`
```
Comment (kbdharun) [inline pages/common/ffplay.md:16]:
```suggestion
`ffmpeg -i {{path/to/file}} -c {{copy}} -f {{media_format}} - | ffplay -`
```
I would suggest using the snake case here as mentioned in [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming)

File: pages/common/ffplay.md [hunk 3]
```diff
@@ -7,6 +7,14 @@
 
 `ffplay {{path/to/file}}`
 
+- Play audio from a media file without a GUI:
+
+`ffplay -nodisp {{path/to/file}}`
+
+- Play media passed by `ffmpeg` through standard stream:
```
Comment (acuteenvy) [inline pages/common/ffplay.md:14]:
```suggestion
- Play media passed by `ffmpeg` through `stdin`:
```

**Comment by Managor** [comment]
Is there an equivalent to `stdin` for standard stream? Feels kinda wrong to say `stdin` or `stdout` in this context

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**